### PR TITLE
Add 9 icons

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
```
    <icon drawable="@drawable/group_alarm" package="com.groupalarm.app" name="GroupAlarm" />
    <icon drawable="@drawable/home_assistant" package="io.homeassistant.companion.android.minimal" name="Home Assistant" />
    <icon drawable="@drawable/home_assistant" package="io.homeassistant.companion.android" name="Home Assistant" />
    <icon drawable="@drawable/teamnspeak" package="com.teamspeak.ts3client" name="TeamSpeak" />
    <icon drawable="@drawable/valetudo" package="cloud.valetudo.companion" name="Valetudo Companion" />
    <icon drawable="@drawable/eventim" package="de.eventim.mobile.app.Android" name="EVENTIM" />
    <icon drawable="@drawable/freenet_mobilfunk" package="de.md.meinmd" name="freenet Mobilfunk" />
    <icon drawable="@drawable/proxmox" package="com.proxmox.app.pve_flutter_frontend" name="Proxmox" />
    <icon drawable="@drawable/rock_am_ring" package="com.mlk.rockamring" name="RAR &amp; RIP" />
    <icon drawable="@drawable/wacken" package="com.wacken" name="Wacken" />
```

### CODE REVIEW

**Comment by daywalk3r666** [comment]
You made mistake. remane your xml to valetudo

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -925,6 +928,7 @@
     <icon drawable="@drawable/talon" package="com.klinker.android.twitter_l" name="Talon for Twitter" />
     <icon drawable="@drawable/tapatalk_pro" package="com.quoord.tapatalk" name="Tapatalk Pro" />
     <icon drawable="@drawable/tasks" package="org.tasks" name="Tasks" />
+    <icon drawable="@drawable/teamnspeak" package="com.teamspeak.ts3client" name="TeamSpeak" />
```
Comment (daywalk3r666) [inline app/src/main/res/xml/grayscale_icon_map.xml:931]: You have a typo here

**Review by Chefski — CHANGES_REQUESTED** [review]
Hey! Got some small tweaks for 3 icons and 1 fix in the `grayscale_icon_map.xml`

Could you fix point on the G in `group_alarm.svg` to align with the other point from the other shape so that it doesn't cut in.
image

<br>

Could you make `rock_am_ring.svg` have a stroke of 12px or at least the circle and lines outside the circle?

<br>

Could you fix this alignments in `valetudo.svg` to be more smooth? Its a similar problem as in `group_alarm.svg`
image

<br>

Also please change the line `<icon drawable="@drawable/rock_am_ring" package="com.mlk.rockamring" name="RAR & RIP" />` to `<icon drawable="@drawable/rock_am_ring" package="com.mlk.rockamring" name="RAR &amp; RIP" />`. 

I recommend grouping all the shapes and making the height, width or both 148px (Only applies when the stroke is centred for all shapes outside and if its not a square). It'll just make sure the shapes are at the borders of the content area size.

**Comment by peterge1998** [comment]
Okay, I did the changes you asked for @Chefski.
However, I can't create the new XML files, because android studio on my laptop does not show new > vector asset...
I don't know why, I am using the flatpak version on my desktop too...

**Comment by daywalk3r666** [comment]
https://inloop.github.io/svg2android/
you can use this instead of android studio

**Comment by Chefski** [comment]
Also why didn't you set the XML name for `freenet.svg` to `freenet_mobilfunk.xml`?

**Review by Chefski — APPROVED** [review]
Thank you

**Comment by peterge1998** [comment]
> https://inloop.github.io/svg2android/ you can use this instead of android studio

I used it for the RAR icon, which looks scrambled in the latest debug build. I will fix it in my next PR (which shouldnt take that long :D)

**Pull request merged**